test/integration/integration.py

```diff
@@ -17,39 +17,53 @@
 
 class TestIntegration(unittest.TestCase):
     def test_integration(self):
-        rospy.logdebug("Integration Test Starting")
+        rospy.logdebug("Autonomy integration testing starting...")
 
         rospy.init_node("integration_test")
 
-        rospy.loginfo("Integration Test Ready")
+        rospy.loginfo("Ready")
 
-        waypoint_in_world = SE3(position=np.array([-5.5, -5.5, 0.0]))
         waypoints = [
             (
                 Waypoint(tag_id=0, type=WaypointType(val=WaypointType.NO_SEARCH)),
-                waypoint_in_world,
+                SE3(position=np.array([4, 4, 0])),
             ),
+            (
+                Waypoint(tag_id=0, type=WaypointType(val=WaypointType.POST)),
+                SE3(position=np.array([-2, -2, 0])),
+            ),
+            (
+                Waypoint(tag_id=1, type=WaypointType(val=WaypointType.POST)),
+                SE3(position=np.array([11, -10, 0])),
+            )
         ]
-
-        publish_waypoints([convert_waypoint_to_gps(waypoint) for waypoint in waypoints])
+        publish_waypoints(list(map(convert_waypoint_to_gps, waypoints)))
```

Comment:
um acshually using map over list comprehension is unpythonic

Reply:
🤓